pkg/common-controller/groupsnapshot_controller_helper.go

```diff
@@ -1396,7 +1396,7 @@ func (ctrl *csiSnapshotCommonController) getGroupSnapshotDriverName(vgs *crdv1al
 
 	// Dynamic groupsnapshots will have a groupsnapshotclass with a driver
 	if vgs.Spec.VolumeGroupSnapshotClassName != nil {
-		class, err := ctrl.getSnapshotClass(*vgs.Spec.VolumeGroupSnapshotClassName)
+		class, err := ctrl.getGroupSnapshotClass(*vgs.Spec.VolumeGroupSnapshotClassName)
```

Comment:
Not related to this but can you please also fix the log in line 1374 from `getSnapshotDriverName: VolumeSnapshot` to `getGroupSnapshotDriverName: volumeGroupSnapshot` as well?